GRAMMAR.md

```diff
@@ -111,6 +111,7 @@ recipe        : attributes* '@'? NAME parameter* variadic? ':' dependency* body?
 attributes    : '[' attribute* ']' eol
 
 attribute     : NAME ( '(' string ')' )?
+              | NAME ':' string
```

Comment:
I'd prefer to leave this syntax for a follow up PR, to keep this one simple.

Reply:
Removed the code that did this parsing from the PR